win32ss/gdi/ntgdi/freetype.c

```diff
@@ -7062,6 +7063,22 @@ IntExtTextOutW(
         bitSize.cx = realglyph->bitmap.width;
         bitSize.cy = realglyph->bitmap.rows;
 
+        /* Do characters other than space return a bitSize.cx of zero? */
+        if (ch0 != 32 && bitSize.cx == 0)
+            DPRINT1("WARNING: WChar 0x%04x has a bitSize.cx of zero\n", ch0);
+
+        /* Don't ignore spaces when computing offset.
+         * This completes the fix of CORE-11787.*/
+        if ((pdcattr->flTextAlign & TA_UPDATECP) && ch0 == 32 && bitSize.cx == 0)
```

Comment:
```suggestion
        if (ch0 != L' ' && bitSize.cx == 0)
            DPRINT1("WARNING: WChar 0x%04x has a bitSize.cx of zero\n", ch0);

        /* Don't ignore spaces when computing offset.
         * This completes the fix of CORE-11787.*/
        if ((pdcattr->flTextAlign & TA_UPDATECP) && ch0 == L' ' && bitSize.cx == 0)
```
Suggesting to replace 32 with the actual wide char.

Reply:
Changed. Thanks.